<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>547</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ベッドモニタリングツール</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="minimumSize">
    <size>
     <width>700</width>
     <height>500</height>
    </size>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QTabWidget" name="tabWidget">
        <property name="currentIndex">
         <number>3</number>
        </property>
        <widget class="QWidget" name="tabPillowSensor">
         <attribute name="title">
          <string>枕センサ</string>
         </attribute>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="PlotWidget" name="graphicsView_acc_x"/>
            </item>
            <item>
             <widget class="PlotWidget" name="graphicsView_acc_y"/>
            </item>
            <item>
             <widget class="PlotWidget" name="graphicsView_acc_z"/>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tabBedSensor">
         <attribute name="title">
          <string>ベッドセンサ</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_5">
              <item>
               <widget class="PlotWidget" name="graphicsView_ch1"/>
              </item>
              <item>
               <widget class="PlotWidget" name="graphicsView_ch2"/>
              </item>
              <item>
               <widget class="PlotWidget" name="graphicsView_ch3"/>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_4">
              <item>
               <widget class="PlotWidget" name="graphicsView_ch4"/>
              </item>
              <item>
               <widget class="PlotWidget" name="graphicsView_ch5"/>
              </item>
              <item>
               <spacer name="verticalSpacer">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>40</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tabStatus">
         <attribute name="title">
          <string>離床モニタリング</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <item>
           <widget class="QTextEdit" name="textEdit_status"/>
          </item>
          <item>
           <widget class="QTextEdit" name="textEdit_log"/>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tabSetting">
         <attribute name="title">
          <string>設定</string>
         </attribute>
         <widget class="QCheckBox" name="checkBox_message">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>30</y>
            <width>191</width>
            <height>22</height>
           </rect>
          </property>
          <property name="accessibleName">
           <string/>
          </property>
          <property name="text">
           <string>担当者へ通知する</string>
          </property>
         </widget>
         <widget class="QPushButton" name="pushButton_reset">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>130</y>
            <width>161</width>
            <height>27</height>
           </rect>
          </property>
          <property name="text">
           <string>一時情報をリセットする</string>
          </property>
         </widget>
         <widget class="QComboBox" name="comboBox_message">
          <property name="geometry">
           <rect>
            <x>40</x>
            <y>60</y>
            <width>141</width>
            <height>27</height>
           </rect>
          </property>
          <item>
           <property name="text">
            <string>電話</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>SMS</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Eメール</string>
           </property>
          </item>
         </widget>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>700</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QGraphicsView</extends>
   <header>pyqtgraph</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
